pysph/solver/application.py

```diff
@@ -1236,6 +1237,24 @@ def _write_info(self, filename, **kw):
         info.update(kw)
         json.dump(info, open(filename, 'w'))
 
+    def _log_solver_info(self, solver):
+        sep = '-'*70
+        eqn_info = '[\n' + ',\n'.join([str(e) for e in self.equations]) + '\n]'
+        particle_info = '\n  '.join(['%s: %d' % (p.name, len(p.gid)) for p in
+                                    solver.particles])
+
+        kernel_name = solver.kernel.__class__.__name__
+        kernel_info = '%s(dim=%s)' % (kernel_name, solver.dim)
+
+        logger.info(
+            'No of particles:\n%s\n  %s\n%s' % (sep, particle_info, sep)
+        )
+        logger.info('Using kernel:\n%s\n  %s\n%s' % (sep, kernel_info, sep))
+        logger.info(
+            'Using integrator:\n%s\n  %s\n%s' % (sep, solver.integrator, sep)
+        )
+        logger.info('Using equations:\n%s\n%s\n%s' % (sep, eqn_info, sep))
```

Comment:
Is there any reason why these `logger` calls aren't passing in the variables as shown in the Python docs?

```python
logger.info('Using equations:\n%s\n%s\n%s', sep, eqn_info, sep)  # ?
```

Reply:
Oh didn't know that, thanks!

Reply:
Sorry, I should have included a link to the relevant docs.

* https://docs.python.org/3/library/logging.html#logging.debug